src/ducks/transactionBuilder.ts

```diff
@@ -254,10 +288,14 @@ export const useTransactionBuilderStore = create<TransactionBuilderState>(
         });
 
         const { hash } = result;
-        set({
-          transactionHash: hash,
-          isSubmitting: false,
-        });
+
+        // Ignore if another submit started later
+        if (get().requestId === currentRequestId) {
```

Comment:
update description (it's ok if we need to repeat part or most of the other descriptions)